tests/pr_test.go

```diff
@@ -29,18 +29,17 @@ const solutionAgentsDADir = "solutions/agents"
 const agentsKubeconfigDir = "solutions/agents/kubeconfig"
 
 // Current supported regions for Observability instances
-// (NOTE: Disabling some regions temporarily due to known issue in those regions)
 var validRegions = []string{
 	"au-syd",
 	"eu-gb",
-	// "eu-de",
-	// "eu-es",
-	// "jp-osa",
-	// "jp-tok",
-	// "us-south",
-	// "us-east",
-	// "ca-tor",
-	// "br-sao",
+	"eu-de",
+	"eu-es",
+	"jp-osa",
+	"jp-tok",
+	"us-south",
+	"us-east",
+	"ca-tor",
+	"br-sao",
```

Comment:
@iamar7 Not all of these are supported regions for event notifications so one of the tests will fail. You will need to pick only regions that EN supports

Reply:
Removed the region not supported by EN